flavour.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Why is that file (that looks like an example) residing in the main directory without any reference to it? If this is an example, place it in a properly named subdirectory, such as `demo` or `example` and describe the usecase for its content in the _README_. Suggestion: "Find an exemplary chefkoch environment, containing an example recipe, step definition and kitchen configuration, in `demo/`"